orchagent/fdborch.cpp

```diff
@@ -72,31 +83,55 @@ bool FdbOrch::storeFdbEntryState(const FdbUpdate& update)
 
     if (update.add)
     {
-        SWSS_LOG_INFO("Storing FDB entry: [%s, 0x%" PRIx64 "] [ port: %s ]",
-                      entry.mac.to_string().c_str(),
-                      entry.bv_id, entry.port_name.c_str());
-        auto inserted = m_entries.insert(entry);
-
-        SWSS_LOG_DEBUG("FdbOrch notification: mac %s was inserted into bv_id 0x%" PRIx64,
-                        entry.mac.to_string().c_str(), entry.bv_id);
-
-        if (!inserted.second)
+        bool mac_move = false;
```

Comment:
Why do we have to interlace add and move? Can we seperate them out as the caller anyways know if its a move or add?

Reply:
You are right, Caller knows it; but since storeFdbEntryState takes FdbUpdate as a parameter which only has add/del indication;
Kept the function signature same and added the logic; 
Also only CRM-Counter is handled differently in MOVE case so it did not seem complicated either.